src/api/v1/Auth/controller.js

```diff
@@ -1,22 +1,137 @@
 const {
     sendSuccessResp,
     sendFailureResp,
-} = require('../../../utils/response.js');
+} = require("../../../utils/response.js");
+
+const { User, Admin } = require("../../../database/index.js");
+
+const { createToken } = require("../../../services/jwt.js");
+
+const { hash, compareHash } = require("../../../services/encryption.js");
```

Comment:
hash is very strictly related to User and Admin model, so this part should be coupled with respective (both)  models only

Reply:
compare part we need to do where ever needed

Reply:
done